content/en/docs/concepts/cluster-administration/addons.md

```diff
@@ -82,6 +82,8 @@ installation instructions. The list does not try to be exhaustive.
 * [Weave Net](https://www.weave.works/docs/net/latest/kubernetes/kube-addon/)
   provides networking and network policy, will carry on working on both sides
   of a network partition, and does not require an external database.
+* [Spiderpool](https://github.com/spidernet-io/spiderpool) is an underlay and RDMA network solution of 
+  Kubernetes, for bare metal, VM and any public cloud.
```

Comment:
```suggestion
* [Spiderpool](https://github.com/spidernet-io/spiderpool) is an underlay and RDMA
  network solution of Kubernetes, designed for bare metal, VMs, and public clouds.
```
It can be inserted before Weave Net, to keep this list in an alphabetical order

Reply:
done